game/EthnicityPage/ChooseEthnicityStep.gd

```diff
@@ -1,7 +1,9 @@
 extends Control
 
-signal ethnicity_chosen(trait_element, current_ethnicity, bonus_attribute)
+signal ethnicity_chosen(current_ethnicity)
 signal attribute_chosen(bonus_attribute)
+signal trait_chosen(trait_element)
```

Comment:
ethnicity trait

Reply:
changed method connected to signal which made more sense in result